src/main/java/core/mate/academy/service/BuldozerProducer.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BuldozerProducer implements MachineProducer<Bulldozer> {
+
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> buldozerList = new ArrayList<>();
+        for (int i = 0; i < 3; i++) {
```

Comment:
let's create constant for number 3

Reply:
Did it for all bulldozer, excavator, and track.